tests/30rooms/06invite.pl

```diff
@@ -87,6 +87,9 @@ sub inviteonly_room_fixture
          $content->{membership} eq "invite" or
             die "Expected membership to be 'invite'";
 
+         require_json_keys( $event, qw( invite_room_state ));
```

Comment:
Currently awaiting rewriting atop PR #71 which should make this alteration better.